org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/CompletionTests11.java

```diff
@@ -17,6 +17,10 @@
 import junit.framework.Test;
 
 public class CompletionTests11 extends AbstractJavaModelCompletionTests {
+	static {
+		// TESTS_NAMES = new String[] {
+		// "test_members_matching_paramater_name_on_field" };
+	}
```

Comment:
This was kept so that a future developer can easily see how they can run selected tests like in other completion tests. But i will remove the tests name and keep the array empty for future developers.